pnpm-workspace.yaml

```diff
@@ -3,20 +3,22 @@ packages:
   - packages/*
   - packages/*/*
 catalog:
-  "@osdk/api": "~2.4.2"
-  "@osdk/client": "~2.4.2"
-  "@osdk/foundry.admin": "~2.38.0"
-  "@osdk/foundry.pack": "~2.38.0"
-  "@osdk/oauth": "~1.4.0"
-  "@osdk/react": "~0.6.0"
-  "@osdk/shared.client2": "~1.0.0"
-  "@types/node": ^20.19.17
+  '@osdk/api': ~2.4.2
```

Comment:
I noticed quotes in the pnpm-workspace.yaml file keep flipping for me too, we might need to tweak dprint configs

Reply:
Might also be pnpm version differences